src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -31,8 +31,22 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// <see cref="ColumnCopyingEstimator"/> copies the input column to another column named as specified in the parameters of the transformation.
+    /// Estimator creating the <see cref="ColumnCopyingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ### Estimator FAQ
+    /// | Question | Answer |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type. | Any |
+    /// | Output column data type. | The same as the data type in the input column. |
+    ///
```

Comment:
>/// [](start = 4, length = 3)

we also need 'additional nugets' here

Reply:
oh, that is going even on that components that are part of Microsoft.ML nuget? Should we put it only on the ones that are not on the main nuget?

---
In reply to: [275070680](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275070680) [](ancestors = 275070680)

Reply:
|"Required NuGet in addition to Microsoft.ML" | None|

---
In reply to: [275072871](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275072871) [](ancestors = 275072871,275070680)